### ISSUE TITLE
Wrong unit conversion?

### ISSUE THREAD

**Issue by ducakar** [ihsoft/KIS#3]
The SC-62 container shows volume of 1.00 m^3 (= 1 000 000 cm^3) in the editor, but it's not possible to fill it past 100 cm^3.

**Comment by ducakar** [comment]
KIS_Shared:413: I recommend changing it to something like:

```
string text = (volume * 1000).ToString("0.00");
if (showUnit) text += " L";
```

**Comment by enceos** [comment]
I'd use decimals of m^3 instead of cm^3, for example: 0.14 m^3. In reality 1 m^3 = 1 million cm^3

**Comment by KospY** [comment]
Fixed for next version (1.0.1)
(Using L unit instead of m3)
